powerauth-data-adapter/src/main/java/io/getlime/security/powerauth/app/dataadapter/impl/validation/AuthenticationRequestValidator.java

```diff
@@ -79,6 +80,11 @@ public void validate(@Nullable Object o, @NonNull Errors errors) {
             errors.rejectValue("requestObject.password", "login.password.long");
         }
 
+        ValidationUtils.rejectIfEmptyOrWhitespace(errors, "requestObject.organizationId", "login.organizationId.empty");
```

Comment:
Do we need to enforce the organization ID even in case the customer has one organization?